t/31-client.t

```diff
@@ -1,96 +0,0 @@
-# Copyright 2018-2021 SUSE LLC
-# SPDX-License-Identifier: GPL-2.0-or-later
-
-use Test::Most;
-
-use FindBin;
-use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
-use OpenQA::Test::TimeLimit '6';
-
-use Test::Mojo;
-use Test::MockModule;
-use Test::MockObject;
-use Test::Output;
-use Test::Warnings ':report_warnings';
-use OpenQA::WebAPI;
-use OpenQA::Test::Case;
-use OpenQA::Script::Client;
-use OpenQA::Scheduler::Client;
-use OpenQA::WebSockets::Client;
-use Mojo::File qw(tempdir);
-
-subtest 'hostnames configurable' => sub {
-    my $config_dir = tempdir;
```

Comment:
We discussed this morning that these tests shouldn't actually be deleted...
Can you recreate this (without the OpenQA::Script::Client tests)?